cmd/nitro/init_test.go

```diff
@@ -411,7 +411,7 @@ func defaultStylusTargetConfigForTest(t *testing.T) *gethexec.StylusTargetConfig
 	return &targetConfig
 }
 
-func TestOpenInitializeChainDbIncompatibleStateScheme(t *testing.T) {
+func TestOpenInitializeExecutionDbIncompatibleStateScheme(t *testing.T) {
```

Comment:
`TestOpenInitializeExecutionDBIncompatibleStateScheme` here and `openInitializeExecutionDB` below